internal_stock_quant_package/models/stock_quant_package.py

```diff
@@ -1,10 +1,20 @@
 # Copyright 2021 ACSONE SA/NV
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class StockQuantPackage(models.Model):
     _inherit = "stock.quant.package"
 
-    is_internal = fields.Boolean("Internal use?")
+    is_internal = fields.Boolean(
+        compute="_compute_is_internal", store=True, string="Internal use?"
+    )
+
+    @api.depends("package_type_id", "package_type_id.package_carrier_type")
```

Comment:
Usually, we don't want to reconfigure a huge amount of packages in case a package type is reconfigured

```suggestion
    @api.depends("package_type_id")
```

Reply:
Makes sense, made the change and updated the tests to match